short-read-mngs/idseq_utils/idseq_utils/batch_run_helpers.py

```diff
@@ -97,10 +97,10 @@ def _run_batch_job(
         jobDefinition=job_definition,
         containerOverrides={
             "environment": [{"name": k, "value": v} for k, v in environment.items()],
-            # (524288 - 1024) / 2, 524288 = r5d.24xlarge memory
-            #   2 = jobs per instance, 1024 = leftover for other processes
-            "memory": 261632,
-            "vcpus": 48,  # 96 / 2, 96 = r5d.24xlarge vcpus , 2 = jobs per instance
+            # (524288 - 1024) / 4, 524288 = r5d.24xlarge memory
```

Comment:
Do we want to just remove this math now that we are likely moving to other instance types soon? This was taken directly from the old scaling math which was actually based on gsnap/rapsearch CPU saturation. I think this may be irrelevant in this brave new world.

Reply:
Removed, I also think the amount of memory on the r5d.24X instance was wrong